deploy/kubernetes/examples/demo-secret-configuration-file.json

```diff
@@ -0,0 +1,24 @@
+{
+  "demo-system-id1": {

```

Comment:
either add "1" suffix to all values of this storage system, or remove it from here